src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -548,7 +548,7 @@ public function getAwsTemporaryUrl($adapter, $path, $expiration, $options)
     public function getRackspaceTemporaryUrl($adapter, $path, $expiration, $options)
     {
         return $adapter->getContainer()->getObject($path)->getTemporaryUrl(
-            Carbon::now()->diffInSeconds($expiration),
+            app(Factory::class)->now()->diffInSeconds($expiration),
```

Comment:
Filesystem component doesn't load `app()` which is under `Illuminate\Foundation`, nor does it require `illuminate/container`.